images/base/Dockerfile.ubuntu

```diff
@@ -1,15 +1,16 @@
 FROM ubuntu:22.04
 
 SHELL ["/bin/bash", "-c"]
+ENV DEBIAN_FRONTEND noninteractive
 
 # Install the Docker apt repository
 RUN apt-get update && \
-    DEBIAN_FRONTEND="noninteractive" apt-get upgrade --yes && \
-    DEBIAN_FRONTEND="noninteractive" apt-get install --yes ca-certificates
+    apt-get upgrade --yes && \
+    apt-get install --yes ca-certificates && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Does it make sense to do this after every apt invocation, or only once at the end?

Reply:
As per https://github.com/hadolint/hadolint/wiki/DL3009 we should do it for each layer i.e each `RUN`, `COPY` or `ADD` command.

Reply:
Also as per Docker [docs](https://docs.docker.com/develop/develop-images/instructions/#apt-get)

> In addition, when you clean up the apt cache by removing /var/lib/apt/lists it reduces the image size, since the apt cache isn't stored in a layer. Since the RUN statement starts with apt-get update, the package cache is always refreshed prior to apt-get install.

> Official Debian and Ubuntu images [automatically run apt-get clean](https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105), so explicit invocation is not required.